docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md

```diff
@@ -76,6 +76,16 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 8 GB memory
 - 200 GB disk space
 
+## Supported YugabyteDB versions
+
+YugabyteDB Anywhere aims to support _all_ officially supported versions of YugabyteDB.
+
+YBA {{< yb-version version="stable" format="short">}} supports deploying the following versions of YugabyteDB:
+
+- v2.14 series: 2.14.0 to 2.14.15
+- v2.18 series: 2.18.0 to 2.18.5
+- v2.20 series: 2.20.0 to 2.20.1.3
+
```

Comment:
add an xref to https://docs.yugabyte.com/stable/yugabyte-platform/manage-deployments/upgrade-software/

"For information on managing YugabyteDB releases and upgrading universes, refer to [Upgrade the YugabyteDB software](../../manage-deployments/upgrade-software/)."